contrib/IECoreMantra/src/IECoreMantra/procedural/Procedural.cpp

```diff
@@ -112,9 +112,13 @@ class VRAY_ieProcedural : public ProceduralPrimitive {
 	virtual int	  initialize(const UT_BoundingBox *);
 	virtual void	 getBoundingBox(UT_BoundingBox &box);
 	virtual void	 render();
-	UT_String m_className; 
+	UT_StringHolder m_className;
```

Comment:
Shouldn't change the member types mid-version, so lets just put them all in the H16 block if needed.

Also, why does `m_classVersion` need to become an int64?